lib/UnoCore/Targets/Android/Android.uxl

```diff
@@ -219,6 +219,38 @@
 
     <!-- Icons -->
 
+    #if @(Project.Android.Icons.AdaptiveIconEnabled:Test(1, 0))
+
+    <ImageFile Name="@(Project.Android.Icons.LDPI:Path || '@//Assets/DefaultIcon.png')"
+        TargetWidth="36" TargetName="app/src/main/res/mipmap-ldpi/icon.png" />
+    <ImageFile Name="@(Project.Android.Icons.MDPI:Path || '@//Assets/adaptive_icon/mipmap-mdpi/icon.png')"
+        TargetWidth="48" TargetName="app/src/main/res/mipmap-mdpi/icon.png" />
+    <ImageFile Name="@(Project.Android.Icons.HDPI:Path || '@//Assets/adaptive_icon/mipmap-hdpi/icon.png')"
+        TargetWidth="72" TargetName="app/src/main/res/mipmap-hdpi/icon.png" />
+    <ImageFile Name="@(Project.Android.Icons.XHDPI:Path || '@//Assets/adaptive_icon/mipmap-xhdpi/icon.png')"
+        TargetWidth="96" TargetName="app/src/main/res/mipmap-xhdpi/icon.png" />
+    <ImageFile Name="@(Project.Android.Icons.XXHDPI:Path || '@//Assets/adaptive_icon/mipmap-xxhdpi/icon.png')"
+        TargetWidth="144" TargetName="app/src/main/res/mipmap-xxhdpi/icon.png" />
+    <ImageFile Name="@(Project.Android.Icons.XXXHDPI:Path || '@//Assets/adaptive_icon/mipmap-xxxhdpi/icon.png')"
+        TargetWidth="192" TargetName="app/src/main/res/mipmap-xxxhdpi/icon.png" />
+    <CopyFile Condition="Android" Name="@(Project.Android.Icons.ANYDPI_v26:Path || '@//Assets/adaptive_icon/mipmap-anydpi-v26/icon.xml')" TargetName="app/src/main/res/mipmap-anydpi-v26/icon.xml" />
+
+    <ImageFile Name="@(Project.Android.Icons.MDPI_Round:Path || '@//Assets/adaptive_icon/mipmap-mdpi/icon_round.png')"
+        TargetWidth="48" TargetName="app/src/main/res/mipmap-mdpi/icon_round.png" />
+    <ImageFile Name="@(Project.Android.Icons.HDPI_Round:Path || '@//Assets/adaptive_icon/mipmap-hdpi/icon_round.png')"
+        TargetWidth="72" TargetName="app/src/main/res/mipmap-hdpi/icon_round.png" />
+    <ImageFile Name="@(Project.Android.Icons.XHDPI_Round:Path || '@//Assets/adaptive_icon/mipmap-xhdpi/icon_round.png')"
+        TargetWidth="96" TargetName="app/src/main/res/mipmap-xhdpi/icon_round.png" />
+    <ImageFile Name="@(Project.Android.Icons.XXHDPI_Round:Path || '@//Assets/adaptive_icon/mipmap-xxhdpi/icon_round.png')"
+        TargetWidth="144" TargetName="app/src/main/res/mipmap-xxhdpi/icon_round.png" />
+    <ImageFile Name="@(Project.Android.Icons.XXXHDPI_Round:Path || '@//Assets/adaptive_icon/mipmap-xxxhdpi/icon_round.png')"
+        TargetWidth="192" TargetName="app/src/main/res/mipmap-xxxhdpi/icon_round.png" />
+    <CopyFile Condition="Android" Name="@(Project.Android.Icons.ANYDPI_v26_Round:Path || '@//Assets/adaptive_icon/mipmap-anydpi-v26/icon_round.xml')" TargetName="app/src/main/res/mipmap-anydpi-v26/icon_round.xml" />
+
+    <CopyFile Condition="Android" Name="@(Project.Android.Icons.foreground:Path || '@//Assets/adaptive_icon/drawable/icon_foreground.xml')" TargetName="app/src/main/res/drawable/icon_foreground.xml" />
```

Comment:
`Condition="Android"` is redundant and has no effect. I suggest to remove this attribute.